lib/rjgit_adapter/git_layer_rjgit.rb

```diff
@@ -30,13 +30,14 @@ def self.sha?(str)
     end
     
     class Actor
-      
-      attr_accessor :name, :email
+
+      attr_accessor :name, :email, :time
       attr_reader :actor
-      
-      def initialize(name, email)
+
+      def initialize(name, email, time = nil)
         @name = name
         @email = email
+        @time = time
         @actor = RJGit::Actor.new(name, email)
```

Comment:
Don't we need to add `time` to the `RJGit::Actor.new` call? Of course that wouldn't work yet, we still need to add that functionality to RJGit.

Reply:
I've opened a PR to let `RJGit::Actor.new` take an optional third `time` (`Time` object expected) argument  [here](https://github.com/repotag/rjgit/pull/54/files#diff-9bf38a62fbfde01973dff41e863a41edR21). The third argument may be `nil`. @Dahie could you modify this PR accordingly?

Reply:
Thanks! Sorry to bother again, but to make this work we need to depend on the latest RJGit (`v5.7.0.1`). Could you update the RJGit dependency [here](https://github.com/gollum/rjgit_adapter/blob/master/gollum-rjgit_adapter.gemspec#L15)?

Reply:
here you go, dependency updated and tests are running